geomesa-spark/geomesa-spark-core/src/main/resources/geomesa/spark.py

```diff
@@ -0,0 +1,39 @@
+from py4j.java_gateway import java_import
```

Comment:
Have we thought about producing a pip package? Refactor the python stuff into a new module, that wraps this up as a pip project. We can include the accumulo-runtime jar in the pip (I think), and so we can distribute everything except for the SPARK dist in one easy to use package.